codalab/worker/worker_run_state.py

```diff
@@ -205,7 +215,16 @@ def mount_dependency(dependency, shared_file_system):
             #   dependency_path:docker_dependency_path:ro
             docker_dependencies.append((dependency.parent_path, dependency.docker_path))
 
+        log_bundle_transition(run_state.bundle.uuid, run_state.stage, RunStage.PREPARING, '')
+
         if run_state.is_killed or run_state.is_restaged:
+            log_bundle_transition(
+                run_state.bundle.uuid,
+                run_state.stage,
+                RunStage.CLEANING_UP,
+                'the bundle is either killed or restaged.',
```

Comment:
I think we can capitalize the sentence and put a colon before stating the reason.